nfts/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "nfts",
+  "version": "1.0.0",
+  "description": "End to End tutorials for NFT deployment and minting",
+  "main": "scripts/index.js",
+  "scripts": {
+    "deployAndMint": "node scripts/deployAndMintNFT.js",
```

Comment:
I belive need to update this . also rename the file

Reply:
done